### PR TITLE
Fix confirmations button positioning.

### LINKED ISSUE #815: UI/UX: Confirmations button positioning has changed
Before:
Screen Shot 2021-09-25 at 16 34 10

Currently:
Screen Shot 2021-09-25 at 16 35 22

### PR DESCRIPTION
**Pull request by MiguelMedeiros**
Fix #815 .

**Observation:** Maybe we should check with @TechMiX about the RTL translations.

### CODE REVIEW

File: frontend/src/app/components/transaction/transaction.component.html [hunk 1]
```diff
@@ -8,9 +8,9 @@
         <span class="d-none d-lg-inline">{{ rbfTransaction.txid }}</span>
       </a>
     </div>
-  
+
     <div>
-      <div class="title float-left">
```
Comment (TechMiX) [inline frontend/src/app/components/transaction/transaction.component.html:13]: We should keep this to maintain the RTL hint

File: frontend/src/app/components/transaction/transaction.component.html [hunk 2]
```diff
@@ -22,7 +22,7 @@ <h1 i18n="shared.transaction">Transaction</h1>
         <app-clipboard [text]="txId"></app-clipboard>
       </div>
 
-      <div class="container-buttons float-right">
+      <div class="container-buttons">
```
Comment (TechMiX) [inline frontend/src/app/components/transaction/transaction.component.html:25]: Add text-right to classes for and RTL hinting.

File: frontend/src/app/components/transaction/transaction.component.html (hunk 1 above)
Comment (MiguelMedeiros) [reply frontend/src/app/components/transaction/transaction.component.html:13]: I've solve this by adding the classes under the `.rtl-layout`.

File: frontend/src/app/components/transaction/transaction.component.html (hunk 2 above)
Comment (MiguelMedeiros) [reply frontend/src/app/components/transaction/transaction.component.html:25]: I've solve this by adding the classes under the `.rtl-layout`.

**Review by wiz — APPROVED** [review]
Tested ACK @ d0a85091

**Pull request merged**